kitchen.yml

```diff
@@ -1,6 +1,7 @@
 ---
 driver:
   name: vagrant
+  flavor_ref: "m1.medium"
```

Comment:
This doesn't do anything for vagrant. Please remove.

Reply:
Hmm. I get OOM without that line. ```[  459.325324] Out of memory: Killed process 2138 (platform-python) total-vm:571200kB, anon-rss:520324kB, file-rss:128kB, shmem-rss:0kB, UID:0 pgtables:1160kB oom_score_adj:0```

Reply:
This is likely due to the fact we're in this transition between how we're using the kitchen CLI. For now, please go ahead and remove it and then update the other two kitchen file. Makes sure you add this in the openstack file.